app/assets/stylesheets/layouts/starry.scss

```diff
@@ -1,85 +1,217 @@
+$bg_color_post_author: #13132F;
+//$bg_color_post_character_narrow: $colors_lightbrown;
+//$bg_color_post_author_narrow: $colors_brown; // these are flipped from not-narrow mode to make the quick-switcher blend in // TODO: Should these flip on starries too?
+$font_color_post_info: #F3F3F3;
+
+.post-character { font-weight: bold; background-color: $bg_color_post_character; }
+.post-character, .post-character a, .post-character a:visited { color: $font_color_post_info; }
+.post-screenname { font-weight: bold; background-color: $bg_color_post_screenname; }
+.post-screenname, .post-screenname a, .post-screenname a:visited { color: $font_color_link_visited; }
+.post-author { font-weight: bold; background-color: $bg_color_post_author; }
+.post-author, .post-author a, .post-author a:visited { color: $font_color_post_info; }
+
+$font_color_paginate_link: #F3F3F3;
+$font_color_page_visited: $font_color_link_visited;
+$bg_color_page_current: #484357;
+$font_color_page_disabled: #58546B;
+
+.pagination .current { background-color: $bg_color_page_current; }
+.pagination a { color: $font_color_paginate_link; }
+.pagination a:visited { color: $font_color_page_visited; }
+.pagination .disabled { color: $font_color_page_disabled; }
+
+/* - icons */
+$border_color_icon_selected: #B9BBEB; // selected icon border
+$bg_color_icon: #D8D8D8;
+$bg_color_icon_credit: #D8D8D8;
+
+.replace-icon, .icons-box { background-color: $bg_color_icon; }
+.icon-credit { background-color: $bg_color_icon_credit; }
+
+#character-icon-selector, .time-loaded { color: #DADADA; }
+
+/* - tag items */
+$bg_color_tag_item: #2C2C5D;
+$font_color_tag_item: #444158;
+$font_color_tag_item_gallery: #F7F7F7;
+
+//TODO: This doesn't have even/odd tag items, but does have gallery-tags tag-item, should that be the case?
+tr.gallery-tags .tag-item { background-color: $bg_color_tag_item; color: $font_color_tag_item_gallery; }
+.tag-item { color: $font_color_tag_item; }
+
+/* - link-boxes */
+$bg_color_linkbox_new: #463C65;
+$bg_color_linkbox_edit: #463C65;
+$bg_color_linkbox_favorite: #463C65;
+$bg_color_linkbox_delete: #463C65;
+$bg_color_linkbox_dismiss: #463C65;
+$bg_color_linkbox_subber: #66608B;
+
+.link-box.action-new { background-color: $bg_color_linkbox_new; }
+.link-box.action-edit { background-color: $bg_color_linkbox_edit; }
+.link-box.action-favorite { background-color: $bg_color_linkbox_favorite; }
+.link-box.action-delete { background-color: $bg_color_linkbox_delete; }
+.link-box.action-dismiss { background-color: $bg_color_linkbox_dismiss; }
+
+.subber .link-box { background-color: $bg_color_linkbox_subber; }
+
+/* - other buttons */
+$bg_color_viewbutton: #463C65;
+$bg_color_viewbutton_selected: #463C65;
+.post-navheader .view-button, .view-button { background-color: $bg_color_viewbutton; }
+.view-button.selected { background-color: $bg_color_viewbutton_selected; }
+
+#character-selector { font-weight: normal; } //TODO: this doesn't seem to exist in starrydark
```

Comment:
it should probably exist in all the starries but I'm not sure; leave this TODO for now and I'll check it out some later point